src/librustc/mir/interpret/mod.rs

```diff
@@ -1,7 +1,7 @@
 //! An interpreter for MIR used in CTFE and by miri
 
 #[macro_export]
-macro_rules! err {
+macro_rules! throw_err {
```

Comment:
I'm calling this `throw_err` and `throw_err_unsup` because I remember you not being ok with the `unsup` word :D. I can change it if needed.